pkg/storage/db.go

```diff
@@ -24,7 +24,8 @@ const (
 	tableAWSHostnames          = "aws_hostnames"
```

Comment:
question: did we intend to break the previous format of putting all the sql queries as constant's above the functions like here https://github.com/asecurityteam/asset-inventory-api/blob/acd88d6f3a3698aa3d778be6caa04a7a9de571c9/pkg/storage/db.go#L85 ? right now we have both styles, sql outside of the functions and sql inside the functions.

Reply:
I would err on the side of constant inside the function if it is not used anywhere else. Does this sound acceptable?

Reply:
discussion we had:

Only SQL that is shared amongst the functions should be declared at the top of the file as a global type variable, SQL only used inside the functions themselves can stay inside the functions.